pkg/server/server.go

```diff
@@ -801,7 +801,7 @@ func validateAdmin(imageSpace string, w http.ResponseWriter, r *http.Request) bo
 	if err != nil {
 		message := fmt.Sprintf("Unable to get permission token %s", err)
 		kiln.LogError.Printf(message)
-		writeErrorResponse(http.StatusUnauthorized, message, w)
```

Comment:
Ok looking at the code in authsdk the two possible errors we can get are:
- `fmt.Errorf("Expected the authorization header to have the format of 'Bearer JWTTOKEN'")`
- ` fmt.Errorf("No 'Authorization' header was found in the request")`